Sources/OfflineEntitlements/PurchasedSK2Product.swift

```diff
@@ -24,6 +23,8 @@ struct PurchasedSK2Product {
 
```

Comment:
I think this should support multiple entitlements correct? It's possible a product has N entitlements

Reply:
yep, that's a bug

Reply:
I'll update

Reply:
That's done later through the entitlement mapping.